### ISSUE TITLE
Issues with emoji

### ISSUE THREAD

**Issue by jcubic** [jcubic/jquery.terminal#514]
### Expected behavior

Emoji is correct

### Actual behavior

Emoji in demo was cut off, and emoji as unicode is marked as 1ch (it should be 2ch).

The fix for css:
```css
.cmd .emoji, .terminal-output>:not(.raw) .emoji {
    background-size: contain;
    background-repeat: no-repeat;
}
```

and solution for width, the formatter need to add style attribute.

**Comment by jcubic** [comment]
It's seems that there is only unicode character with 1ch that I was using, rest the of emoji have 2 characters size.

**Comment by jcubic** [comment]
released in 2.8.0
